### PR TITLE
BUGFIX: Missing empty attributes.

### PR DESCRIPTION
**Pull request by rjanjic**
PLEASE MAKE SURE YOU READ [CONTRIBUTING GUIDELINES](../CONTRIBUTING.md) and [CODE OF CONDUCT](../CODE_OF_CONDUCT.md).

UNLESS IT IS A VERY SMALL CHANGE, CREATE AN ISSUE FIRST FOR DISCUSSION.
** DON'T FORGET TO ADD YOUR CHANGES TO [CHANGELOG.md](../CHANGELOG.md) under
proper category **

i.e:

```
## master

### Features

- [#ISSUEID](https://github.com/trivago/melody/issues/000) Added coffee brewing feature
```

REMOVE THIS AND EDIT THE TEMPLATE BELOW

---

#### What changed in this PR:

Write the description and changes here...

### CODE REVIEW

File: packages/melody-compiler/__tests__/__fixtures__/success/empty_attribute.template [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div itemscope itemtype="http://schema.org/Hotel">
```
Comment (pago) [inline packages/melody-compiler/__tests__/__fixtures__/success/empty_attribute.template]: I'd really like to see end to end tests for this to see which attributes will work that way and which won't.

**Review by pago — APPROVED** [review]

**Comment by pago** [comment]
@uzi88 We'll also need a change to the `CHANGELOG.md` to document what you've changed.

**Comment by iandevlin** [comment]
> 
> 
> @iandevlin could you also take a look at this change? It'll effectively call `el.setAttribute('itemscope', true)` for all attributes which are specified without a value.

@pago And what will that result in? Setting `itemscope="true"`? (as this is invalid)

**Comment by pago** [comment]
For some boolean attributes would be set. i.e. for `itemscope` it'll result in `<div itemscope></div>`. I was wondering if you happen to know about attributes for which this wouldn't hold true and which would result in the `foo="true"` case you mentioned.

**Comment by iandevlin** [comment]
Why am I seeing all this stuff so late...

**Pull request closed without merging**